src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -464,7 +466,7 @@ export function useSwapInputsController({
 
     try {
       const [quoteResponse, fetchedPrices] = await Promise.all([
-        params.swapType === SwapType.crossChain ? getCrosschainQuote(params) : getQuote(params),
+        isCrosschainSwap ? getCrosschainQuote(params) : getQuote(params),
```

Comment:
Could these be consolidated to a single endpoint in the swaps sdk? Always thought it was weird to distinguish between the two on the client-side

Reply:
I think it's possible esp. since so much of the socket-specific code in the app was gutted